Tests/SwiftGenKitTests/TestsHelper.swift

```diff
@@ -155,6 +155,40 @@ func convertToString(_ value: Any) -> String? {
   }
 }
 
+func convertToDictionary(_ value: Any) -> [String: Any]? {
+  switch value {
+  case let value as [String: Any]:
+    return value
+  case let value as NSObject:
+    return convertNSObjectToDictionary(value)
+  default:
+    return nil
+  }
+}
+
+func convertNSObjectToDictionary(_ object: NSObject) -> [String: Any] {
+  var result: [(String, Any)] = []
+  var mirror: Mirror? = Mirror(reflecting: object)
+
+  repeat {
+    // ensure we can property handle lazy variables
+    // we use `object.value(forKey:)` to force load lazy variables
+    result += mirror?.children
+      .compactMap { $0.label?.deletingPrefix("$__lazy_storage_$_") }
+      .map { ($0, object.value(forKey: $0) as Any) } ?? []
```

Comment:
We can't use a `$0.value` (a Mirror child's value), as that evaluates to `nil` and doesn't trigger lazy evaluation.

Cool trick though if you want to check if `lazy` has been evaluated yet.

Reply:
Shouldn't we try to do recursion here? Like if the object's value is itself a dictionary-like thing, recursively call `convertNSObjectToDictionary` on that too?

I mean, currently the only `NSObject` that we have in our contexts is gonna be this top-level `StencilContextLazyDocument` object so that shouldn't really cause any trouble now, but just wondering for the future.

(Also, maybe that kind of conversion better belongs into `Stencil` itself? 🤔 )